### PR TITLE
Update check rounds script

### PR DESCRIPTION
**Pull request by Baiqiang**
1. Remove checking for a non-combined round listed as combined
2. Remove related data while removing events

For 2, @SAuroux I'm not sure if we need to add a confirmation.

### CODE REVIEW

**Comment by SAuroux** [comment]
Thanks @Baiqiang . 

I think the idea with the confirmation is good, please go ahead add one if it is not too complicated.

File: webroot/results/admin/check_rounds.php [hunk 1]
```diff
@@ -612,7 +612,11 @@ function checkEvents () {
       echo "  Old events list: ".implode(' ', $intersect)." <b style='color:#F00'>".implode(' ',$competitionOnly)."</b><br />\n";
       echo "  New events list: ".implode(' ', $intersect)." <b style='color:#3C3'>".implode(' ',$resultsOnly)."</b><br />\n";
       foreach($competitionOnly as $event) {
-        dbCommand("DELETE from competition_events where competition_id='$competitionId' and event_id = '$event'");
+        $competitionEventId = dbValue("SELECT id FROM competition_events WHERE competition_id='$competitionId' AND event_id = '$event'");
+        dbCommand("DELETE FROM competition_events WHERE id=$competitionEventId");
```
Comment (viroulep) [inline webroot/results/admin/check_rounds.php:616]: Should we add an assert/sanity check that `$competitionEventId` is not null/empty?
What happens if it is in that query?

Comment (Baiqiang) [reply webroot/results/admin/check_rounds.php:616]: Hi @viroulep , the `$competitionOnly` comes from https://github.com/thewca/worldcubeassociation.org/blob/84129d07c4475164bdd2f552c1e25d4abacbd922/webroot/results/admin/check_rounds.php#L586 so I don't think it needs assert.

Comment (viroulep) [reply webroot/results/admin/check_rounds.php:616]: Ok thanks for the pointer!
I tried running the `DELETE` with an empty id and it seems the worst that can happen is a syntax error anyway.

**Comment by Baiqiang** [comment]
@SAuroux  Here are the screenshot for confirmations of removing and adding events:

image
image

**Comment by SAuroux** [comment]
This looks great, thanks @Baiqiang

**Comment by Baiqiang** [comment]
@viroulep I've squashed the commits into one.

**Pull request merged**